src/modules/flight_mode_manager/tasks/Orbit/FlightTaskOrbit.cpp

```diff
@@ -329,8 +331,12 @@ void FlightTaskOrbit::_generate_circle_setpoints()
 	Vector3f center_to_position = _position - _center;
 	// xy velocity to go around in a circle
 	Vector2f velocity_xy(-center_to_position(1), center_to_position(0));
+
+	// slew rate is used to reduce the jerk when starting an orbit.
+	_slew_rate_velocity.update(_orbit_velocity, _deltatime);
+
 	velocity_xy = velocity_xy.unit_or_zero();
-	velocity_xy *= _orbit_velocity;
+	velocity_xy *= _slew_rate_velocity.getState();
```

Comment:
Also need to use this value in the centripetal acceleration feedforward a couple of lines below instead of `_orbit_velocity`

Reply:
yea, i seem to have overlooked that. thanks for catching it.